### PR TITLE
Adds Ross

### PR DESCRIPTION
**Pull request by rosschapman**

### CODE REVIEW

File: src/about-us/index.json [hunk 1]
```diff
@@ -26,6 +26,19 @@
          "specialties": "Software Engineering: Backend, APIs, Infrastructure, Automation, Engineering Leadership",
          "interests": "Democratic Governance, Facilitation, Books, Cats, Knitting, Food",
          "bio": "Ana is an experienced software engineer and engineering leader. She's worked at companies big and small, as an employee and as a contractor, as an individual contributor and as a manager. She stumbled upon Zinc in her search for better ways of working together, which is her current focus. Her engineering strengths lie in the backend, services, APIs, infrastructure and automation, as well as process and leadership coaching."
+      },
+      {
+         "name": "Ross CHapman",
```
Comment (anaulin) [inline src/about-us/index.json]: s/CHapman/Chapman/

**Review by anaulin — APPROVED** [review]
Feel free to merge whenever you're ready. Once you merge, it will be auto-deployed to the production.

File: .eleventy.js [hunk 2]
```diff
@@ -11,6 +13,10 @@ module.exports = function(config) {
 
   config.addWatchTarget("./src/styles/");
 
+  config.addPairedNunjucksShortcode("markdown", (content) => {
+    return new markdownIt({ html: true }).render(content);
+  });
```
Comment (rosschapman) [inline .eleventy.js:18]: I'm not sure why this is necessary but I suspect that the render sequencing is a bit funky when rendering out data from a json file.

File: src/about-us/index.json (hunk 1 above)
Comment (rosschapman) [inline src/about-us/index.json]:
```suggestion
         "name": "Ross Chapman",
```

**Comment by rosschapman** [comment]
@anaulin I don't have owner privileges on this repo to merge. Can you add me as owner for the org, please?

**Comment by anaulin** [comment]
@rosschapman it looks like you were already added as an owner for the org by someone else. Can you try again?

**Pull request merged**